custom/icds_reports/templates/icds_reports/icds_app/awc-reports.html

```diff
@@ -1,5 +1,5 @@
 {% load i18n %}
-<div cg-busy="[$ctrl.myPromise, $rootScope.locationPromise]">
+<div cg-busy="[$ctrl.myPromise]">
```

Comment:
do you know when this became irrelevant?

Reply:
The last use I saw was two years ago, but it is possible it existed somewhere after that that I missed.